### PR TITLE
PACT-496: Updated scala version to 2.13.10

### PR DESCRIPTION
**Pull request by userjd**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -33,7 +33,7 @@ lazy val root = Project("kettle-test-framework", file("."))
       )
     ),
     scalacOptions ++= Seq(
-      "-target:jvm-1.8",
+      "-release:8",
```
Comment (rwalpole) [inline build.sbt]: I think this should be `"-release", "8"` - see my comments on https://github.com/nationalarchives/tna-cat/pull/326

**Review by rwalpole — CHANGES_REQUESTED** [review]

**Review by rwalpole — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (userjd) [reply build.sbt]: Done

File: src/test/scala/ExampleWorkflowSpec.scala [hunk 2]
```diff
@@ -103,10 +103,9 @@ class ExampleWorkflowSpec extends AnyWordSpec with Matchers with BeforeAndAfterA
     deleteTestFiles()
   }
 
-  override def afterAll() {
+  override def afterAll(): Unit =
     clearDatabaseDataDir()
-    deleteTestFiles()
-  }
+  deleteTestFiles()
```
Comment (rwalpole) [inline src/test/scala/ExampleWorkflowSpec.scala]: Seems we are missing some curly braces around this function. I think without the curly braces only `clearDatabaseDataDir()` will be called by `afterAll()`

**Review by rwalpole — CHANGES_REQUESTED** [review]

File: src/test/scala/ExampleWorkflowSpec.scala (hunk 2 above)
Comment (userjd) [reply src/test/scala/ExampleWorkflowSpec.scala]: well spotted, bug fixed !

**Review by rwalpole — APPROVED** [review]

**Pull request merged**